shinydashboard/lantern/ui.R

```diff
@@ -84,196 +92,248 @@ ui <- dashboardPage(
+      #endpoint_modal_tabset li a {
+        color: #024A96;
+      }
+
+      /* Change the text color of the active navbar tab on the Organization page, Resource page, Profile page, and Endpoint Modal */
+      .nav-tabs>li.active>a {
         color: #555!important;
       }
-      .multi-wrapper div a {
+   
+      /* Change color of the selected links in the resources checkbox */
+      .multi-wrapper .selected-wrapper .item.selected  {
         color: #024A96;
       }
+
+      /* Change color of the non selected links in the resources checkbox */
       .multi-wrapper .non-selected-wrapper .item.selected {
         color: #4F4F4F;
         opacity: 1!important;
       }
-      a:link {
-        text-decoration: none;
-      }
+      
+      /* Remove underline from a tag elements when they are visted */
       a:visited {
         text-decoration: none;
       }
+
+      /* Bold a tag elements when they are hovered on */
       a:hover {
         font-weight: bold;
       }
+
+      /* Change coloring for FHIR version dropdown */
       button.dropdown-toggle {
         background-color: white!important;
         color: black;
       }
+
+      /* Add a border when a button is hovered over */
       button:hover {
         border: 3px solid!important;
-      }      
+      }     
+
+      /* Add a border and when a select tag (dropdowns) is hovered over and make sure background stays white */
       select:hover {
         border: 3px solid!important;
         background-color: white!important;
       }
+
+      /* Bold an a tag element when it is active */
       a:active {
         font-weight: bold;
       }
+
+      /* Add a border to button when it is active */
       button:active {
         border: 3px solid!important;
-      }      
+      } 
+
+      /* When an a tag is focused on, add a border, change background color to yellow, and change text font to black */
       a:focus-visible  {
         border: 4px solid!important;
         background-color: yellow!important;
         color: black!important;
       }
+
+      /* When a button is focused on, add a border, change background color to yellow, and change text font to black */
       button:focus-visible  {
         border: 4px solid!important;
         background-color: yellow!important;
         color: black!important;
       }
+
+      /* When an select tag (dropdowns) is focused on, add a border, change background color to yellow, and change text font to black */
       select:focus-visible  {
         border: 4px solid!important;
         background-color: yellow;
         color: black!important; 
       }
-      .selectize-input:hover {
-        border: 3px solid!important;
-      }
-      .selectize-input:focus-visible {
-        border: 4px solid!important;
-        background-color: yellow!important;
-        color: black!important; 
-      }
+
+      /* Add a border when an input tag (textbox) is hovered over /*
```

Comment:
On this comment and some of the ones below, the comment ends with `/*` instead of `*/`